packages/angular/src/generators/application/angular-v14/application.ts

```diff
@@ -38,6 +37,7 @@ export async function applicationGenerator(
     skipFormat: true,
   });
 
+  const { wrapAngularDevkitSchematic } = require('@nrwl/devkit/ngcli-adapter');
```

Comment:
I'm assuming this is needed because of some imports occurring in the `ngcli-adapter` that cause issues. Wouldn't it be better to update `ngcli-adapter` to inline the `require`s instead? That way is irrelevant to the consumer how to import it.

Reply:
the issue is that in packages/nx/src/adapter/ngcli-adapter.ts, it extends a class from '@angular-devkit/core'
`export class NxScopedHost extends virtualFs.ScopedHost<any> {`
for this extends a class, could not change the import to inside the function.